### ISSUE TITLE
Sync plugin throws exception for non-pipeline builds

### ISSUE THREAD

**Issue by bparees** [fabric8io/openshift-jenkins-sync-plugin#95]
When running a non-pipeline build from jenkins, this exception appears in the logs:

```
INFO: OpenShift Sample #1 main build action completed: FAILURE
Jul 12, 2016 1:37:24 PM hudson.model.listeners.RunListener report
WARNING: RunListener failed
java.lang.IllegalStateException: Cannot poll a non-workflow run
        at io.fabric8.jenkins.openshiftsync.BuildSyncRunListener.pollRun(BuildSyncRunListener.java:171)
        at io.fabric8.jenkins.openshiftsync.BuildSyncRunListener.onCompleted(BuildSyncRunListener.java:141)
        at hudson.model.listeners.RunListener.fireCompleted(RunListener.java:201)
        at hudson.model.Run.execute(Run.java:1783)
        at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
        at hudson.model.ResourceController.execute(ResourceController.java:98)
        at hudson.model.Executor.run(Executor.java:410)
```

I don't think we should be reporting that error.

@jimmidyson

**Comment by jimmidyson** [comment]
Released in 0.0.11 - available from https://repo.jenkins-ci.org/releases/io/fabric8/jenkins/plugins/openshift-sync/0.0.11/

**Comment by bparees** [comment]
thanks @jimmidyson!
